posts/when_to_fold_em.md

```diff
@@ -0,0 +1,48 @@
+# When to fold 'em
+
+Some years ago, I worked as part of a development team on a project to upgrade a tangled legacy system managing a company’s payment systems. Two months in, and we had very little to show for it - pages of diagrams that looked like spiderwebs, a few outages caused by failed attempts to untangle the pile, and a rising level of frustration.
+
+That company had spent thousands of pounds, not to mention missed out on a bunch of other opportunities - and not got a good return on that investment. Stakeholders weren’t any happier than we were, and it was time to decide what to do next.
+
+> We’ve learned some things about what we’re trying to tackle here. Let’s give it another couple of months and see if we can make any progress.
```

Comment:
maybe it's just me, but I was a little confused at first ^^ how can we make it clearer that this is a conversation that happened?

Reply:
Yeah, I agree. Think I'll just introduce these with another sentence initially, then hopefully it's clear for the rest of the article.